element_lab/export/nwb.py

```diff
@@ -1,11 +1,14 @@
 from element_lab import lab
 
 
-def lab_to_nwb_dict(lab_key):
-    """
-    Generate a dictionary containing all relevant lab and institution info
-    :param lab_key: Key specifying one entry in element_lab.lab.Lab
-    :return: dictionary with NWB parameters
+def lab_to_nwb_dict(lab_key: dict) -> dict:
```

Comment:
I'm thinking these first 3 should be `_Prefixed` so as to be hidden in API docs. I don't think the average user needs separate funcs. --- Just a note for a future PR

Reply:
ok, I'll make that change.